.eslintrc.yml

```diff
@@ -32,7 +32,12 @@ overrides:
       ecmaVersion: 2017
     env:
       browser: no
-
+  - files:
+      - esm-utils.js
+    parserOptions:
+      ecmaVersion: 2018
+      sourceType: module
+    parser: babel-eslint
```

Comment:
- shouldn't we add: `env: browser: false` here?
- do we still need _babel-eslint_, despite our _esLint_ update?